csv/generic/src/fs2/data/csv/generic/CsvName.scala

```diff
@@ -0,0 +1,9 @@
+package fs2.data.csv.generic
+
+import scala.annotation.Annotation
+
+/**
+ * Mark an object of sealed trait to have a different representation in CSV than the name of the type.
+ * @param name the name to expect in the CSV
+ */
+case class CsvName(name: String) extends Annotation
```

Comment:
Wouldn't `CsvValue` be a better fit. With `Name` I first understood _header name_.

Reply:
Agree, `CsvValue` is a better name, so I just changed it.